### DISCUSSION TITLE
Is it possible to delete gift cards with a "test mode" flag?

### DISCUSSION THREAD

**Discussion by c0de-bender** [pretix/pretix#3774] [Q&A] [answered]
When manually issuing a gift card for an organizer, I can check the option "Test mode card". Also, when – for an event in test mode – I make an order of a product which has the checkbox "This product is a gift card" in its product settings checked, the gift card created after payment is a test mode card too.

Is there a way to delete those test gift cards (similar to the option to delete test orders)?
If not, would you want to have this feature? If so, could I try to implement this? I'd like to start with something easy and this looks easy enough (from my naïve perspective).

**Comment by raphaelm** [comment, marked as answer]
> Is there a way to delete those test gift cards (similar to the option to delete test orders)?

No.

> If not, would you want to have this feature? If so, could I try to implement this? I'd like to start with something easy and this looks easy enough (from my naïve perspective).

I'm not sure, I'm leaning towards no. The data model around gift cards does not give us strong guarantees the gift card is not "used" anywhere and e.g. deleting a gift card without deleting an order where it was used might lead to funny bugs.

**Reply by c0de-bender** [reply]
Ok, thanks for the quick feedback! :slightly_smiling_face:
